js/apps/account-ui/src/account-security/SigningIn.tsx

```diff
@@ -19,12 +19,10 @@ import {
 } from "@patternfly/react-core/deprecated";
 import { CSSProperties, useState } from "react";
 import { Trans, useTranslation } from "react-i18next";
-import { ContinueCancelModal, useAlerts } from "ui-shared";
-import { deleteCredentials, getCredentials } from "../api/methods";
```

Comment:
The `deleteCredentials()` function is never used anywhere else, so it can be removed from the Account UI.

Reply:
@jonkoops Updated and removed `deleteCredentials` from `methods.ts`. Thanks for pointing this!